src/components/EventCalendar/EventCalendarListPanel.tsx

```diff
@@ -179,10 +181,16 @@ const EventCalendarListPanel = ({ initialRequiredLabelIds, initialOptionalLabelI
                             {events && requiredLabels && optionalLabels && (
                                 <WidthFixer className={loading ? classes.loading : undefined}>
                                     <div className={classes.iCal}>
-                                        {t('EventCalendar.gCalText')}
-                                        <a href={t('EventCalendar.gCalUrl')} target="_blank" rel="noreferrer">
+                                        {/* eslint-disable-next-line react/style-prop-object */}
```

Comment:
Když ti na něco řve lint, je to proto, že bys to neměl dělat ;) Buď na ten odkaz vytvoř další styl. nebo do stylu iCal přidej:

```
'& a': {
  fontWeight: 'bold',
},
```

Reply:
Point, řvoucího linteru jsem si nevšmil - přidává se tam automaticky tím fixem?

Každopádně jsem to opravil, jak říkáš. Díky.